### ISSUE TITLE
Sphere getting replaced by tiny RCC

### ISSUE THREAD

**Issue by ljacobson64** [svalinn/mcnp2cad#22]
I'm trying to run mcnp2cad on the following input file:

```
1     1  -8.00   -1      imp:p 1.
2     0           1  -2  imp:p 1.
3     0               2  imp:p 0.

1     rcc  0. 0. -.0001 0. 0. .0002 .0001
2     so   10
```

What happens is that the sphere doesn't exist in the .sat file and it is instead replaced with a copy of the tiny RCC. The problem still appears when I remove cell 3, and it still appears if I use mcnp2cad -t 1e-8.

The problem disappears if I make the RCC bigger, and it also disappears if I run mcnp2cad -G to prevent creation of the graveyard.

Any ideas on why this would happen or how to fix it?

**Comment by ljacobson64** [comment]
Ignore this... I was using an old commit to it to build with CGM 12.2, but this issue appears to have been fixed since then.

**Comment by ljacobson64** [comment]
To clarify, the issue was fixed between CGM 12.2 and 13.1 and has nothing to do with mcnp2cad.
